charts/midaz/templates/bootstrap-mongodb.yaml

```
@@ -0,0 +1,115 @@
+{{- if .Values.global.externalMongoDefinitions.enabled }}
+{{- $creds := .Values.global.externalMongoDefinitions.midazCredentials -}}
+{{- $rolesJSON := toJson $creds.roles -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: midaz-bootstrap-mongodb
+  namespace: {{ .Release.Namespace }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)